src/components/themed/WalletListCurrencyRow.js

```diff
@@ -67,14 +74,14 @@ class WalletListRowComponent extends React.PureComponent<Props> {
 
     if (!exchangeRate || !differencePercentage || isNaN(differencePercentage)) {
       differencePercentageString = ''
-    } else if (exchangeRate && differencePercentage && differencePercentage === 0) {
+    } else if (exchangeRate && differencePercentage && differencePercentage === '0') {
       differencePercentageString = '0.00%'
-    } else if (exchangeRate && differencePercentage && differencePercentage < 0) {
+    } else if (exchangeRate && differencePercentage && parseFloat(differencePercentage) < 0) {
       differencePercentageStyle = 'Negative'
-      differencePercentageString = `-${Math.abs(differencePercentage).toFixed(1)}%`
-    } else if (exchangeRate && differencePercentage && differencePercentage > 0) {
+      differencePercentageString = `-${Math.abs(parseFloat(differencePercentage)).toFixed(1)}%`
+    } else if (exchangeRate && differencePercentage && parseFloat(differencePercentage) > 0) {
       differencePercentageStyle = 'Positive'
-      differencePercentageString = `+${Math.abs(differencePercentage).toFixed(1)}%`
+      differencePercentageString = `+${Math.abs(parseFloat(differencePercentage)).toFixed(1)}%`
```

Comment:
No need to go back to numbers for comparisons, you should use this:

https://www.npmjs.com/package/biggystring#comparisons